experiments/AtmosLES/dycoms.jl

```diff
@@ -326,7 +326,13 @@ function config_dycoms(FT, N, resolution, xmax, ymax, zmax)
     ode_solver = ClimateMachine.ExplicitSolverType(
         solver_method = LSRK144NiegemannDiehlBusch,
     )
-
+    #=
+    ode_solver = ClimateMachine.MISSolverType(;
+        splitting_type = ClimateMachine.SlowFastSplitting(),
+        mis_method = MIS2,
+        fast_method = LSRK144NiegemannDiehlBusch,
+        nsubsteps = 18,
+    )=#
```

Comment:
Did you intend to commit these changes?

Reply:
Ah no I was working on another branch fro those. How did they get in?